rancher/context.go

```diff
@@ -140,10 +152,32 @@ func (c *Context) unmarshalBytes(composeBytes, bytes []byte) error {
 }
 
 func (c *Context) fillInRancherConfig(rawServiceMap config.RawServiceMap) error {
+	var binding BindingProperty
+	var bindingsServices []string
 	if err := config.Interpolate(c.EnvironmentLookup, &rawServiceMap); err != nil {
 		return err
 	}
-	rawServiceMap, err := preprocess.TryConvertStringsToInts(rawServiceMap)
+
+	bindingsContent, err := ioutil.ReadFile("bindings.json")
+	if err != nil {
+		return err
+	}
+	c.BindingsBytes = bindingsContent
+	err = json.Unmarshal(c.BindingsBytes, &binding)
```

Comment:
We're already unmarshaling in `PreprocessServiceMap`, so we shouldn't need to do it here